### ISSUE TITLE
http://www.bioconductor.org/developers/how-to/useDevel/ still references BiocInstaller

### ISSUE THREAD

**Issue by PeteHaitch** [Bioconductor/bioconductor.org#10]
I was making changes requested by @LiNk-NY in https://stat.ethz.ch/pipermail/bioc-devel/2018-July/013815.html. In updating the **GenomicTuples** I saw that I linked to http://www.bioconductor.org/developers/how-to/useDevel/ for instructions on using the devel branch, but these still reference **BiocInstaller** rather than **BiocManager**. Is there a new link I should be using with updated instructions or will this page be updated?

**Comment by LiNk-NY** [comment]
Thanks for pointing this out Pete @PeteHaitch . We'll update this. 
You can check the devel landing page of your package under the **Installation** section: http://bioconductor.org/packages/devel/bioc/html/GenomicTuples.html
Also, I've posted the appropriate code chunk to use in the `bioc-devel` mailing list: 

```r
install.packages("BiocManager")
BiocManager::install("GenomicTuples")
```

**Comment by LiNk-NY** [comment]
I'm CC'ing Lori and Martin. @lshep @mtmorgan 
This looks like a devel-only page so it should be updated.

**Comment by lshep** [comment]
I'm fine with updating the page whenever  @mtmorgan  does this seem reasonable to do?   @LiNk-NY did you want to do a pull request for the updates?

**Comment by mtmorgan** [comment]
yes the page should be updated, no need to wait.

**Comment by lshep** [comment]
@LiNk-NY @mtmorgan    I created a new branch called UpdateUseDevel -  You two know the ins and outs and whats important so can you please review and see if its appropriate esp concerning following the rules of .libPaths -  I assumed to keep the two sections since they reference the different versions of R being used -  and the last section I wasn't sure if they would still need to remove and reinstall BiocManager,  I assume an update.package("BiocManager") might be appropriate enough although I haven't added that line yet - make any changes to the branch and let me know when I can merge it into the master.

**Comment by mtmorgan** [comment]
closed via issue #12
